server/controllers/konnectors.coffee

```diff
@@ -116,4 +118,22 @@ module.exports =
         req.konnector.updateFieldValues { accounts: accounts }, (err) ->
             return next err if err
 
-            res.redirect '../../../#konnector/' + req.konnector.slug
+            res.status(200).send """<!DOCTYPE html>
```

Comment:
Only during the configuration of the konnector : it's the response from the redirect route ; the script just close the window which allows to identify and authenticate.